packages/atomic/src/components/search/facets/atomic-category-facet/atomic-category-facet.tsx

```diff
@@ -133,6 +134,10 @@ export class AtomicCategoryFacet
    * Also determines the number of additional values to request each time more values are shown.
    */
   @Prop({reflect: true}) public numberOfValues = 8;
+  /**
+   * The tabs on which to display the facet.
```

Comment:
@louis-bompart since you were the one that suggested this syntax, what are your thoughts on this? 

Also, since it is related what do you think about Oli's suggestion to use `@ArrayProp`?

Reply:
I'm not that attached to the syntax, I'd trust the DXUI hivemind opinion :)
ArrayProp in any case make sense

Reply:
What do you think of something like this then? More concise than what you suggested, but seems clear enough to me.

```typescript
  /**
   * The tabs on which to display the facet.
   *
   * Specify the property as an array using a JSON string representation:
   * ```html
   *  <atomic-facet tabs-included='["tabA", "tabB"]'></atomic-facet>
   * ```
   * If a tab is specified in both `tabs-included` and `tabs-excluded`, the exclusion takes precedence and the component will not be displayed on that tab.
   */
  @ArrayProp()
  @Prop({reflect: true, mutable: true})
  public tabsIncluded: string[] | string = '[]';

  /**
   * The tabs on which to exclude the facet.
   *
   * Specify the property as an array using a JSON string representation:
   * ```html
   *  <atomic-facet tabs-excluded='["tabA", "tabB"]'></atomic-facet>
   * ```
   * If a tab is specified in both `tabs-included` and `tabs-excluded`, the exclusion takes precedence and the component will not be displayed on that tab.
   */
  @ArrayProp()
  @Prop({reflect: true, mutable: true})
  public tabsExcluded: string[] | string = '[]';
  ```

Reply:
I like it! Here's a counter proposal with some nitpicks...

```typescript
  /**
   * The tabs on which the facet can be displayed. This property complements `tabs-excluded`.
   *
   * Set this property as a stringified JSON array, e.g.,
   * ```html
   *  <atomic-facet tabs-included='["tabIDA", "tabIDB"]'></atomic-facet>
   * ```
   * If you don't set this property, or set it to `'[]'`, the facet can be displayed on any tab. Otherwise, the facet can only be displayed on the specified tabs. In either case, the facet won't be displayed on any of the tabs specified in the `tabs-excluded` property (exclusion takes precedence).
   */
  @ArrayProp()
  @Prop({reflect: true, mutable: true})
  public tabsIncluded: string[] | string = '[]';

  /**
   * The tabs on which this facet must not be displayed. This property complements `tabs-included`.
   *
   * Set this property as a stringified JSON array, e.g.,
   * ```html
   *  <atomic-facet tabs-excluded='["tabIDA", "tabIDB"]'></atomic-facet>
   * ```
   * If you don't set this property, or set it to `'[]'`, the facet can be displayed on any tab. Otherwise, the facet won't be displayed on any of the specified tabs. In either case, the `tabs-included` property can further restrict the tabs on which the facet can be displayed.
   */
  @ArrayProp()
  @Prop({reflect: true, mutable: true})
  public tabsExcluded: string[] | string = '[]';
```